.github/workflows/ci.yml

```diff
@@ -191,11 +193,7 @@ jobs:
         with:
           distribution: 'temurin'
           java-version: 8
-      - name: Execute tests
-        run: hadoop-ozone/dev-support/checks/${{ matrix.check }}.sh
-      - name: Summary of failures
-        run: cat target/${{ matrix.check }}/summary.txt
-        if: ${{ !cancelled() }}
```

Comment:
This part should not be removed, otherwise these basic checks become no-ops.

Reply:
@adoroszlai  thanks for review, I have a question here.. If we'll not remove step -> `Execute tests`, then unit.sh (unit tests) will run as part of basic check and we want to separate out unit test run out of basic checks...? Did I understood anything wrong here ?

Reply:
I think you had already fixed this problem by the changes done in `selective_ci_checks.sh`.  `unit` is no longer emitted as part of the list of basic checks.

Reply:
> I think you had already fixed this problem by the changes done in `selective_ci_checks.sh`. `unit` is no longer emitted as part of the list of basic checks.

Yes, so we should not include below lines in basic checks...right ?
```

      - name: Execute tests
        run: hadoop-ozone/dev-support/checks/${{ matrix.check }}.sh
      - name: Summary of failures
        run: cat target/${{ matrix.check }}/summary.txt
        if: ${{ !cancelled() }}
```

Reply:
No, we should include these.  They are required for all checks (`checkstyle`, `bats`, etc.), not only `unit`.  So the current state of the patch is fine.

Reply:
Yes you are right. I got it what you meant. Thanks for correcting me.